### PR TITLE
Core Beta release

### PR DESCRIPTION
**Pull request by asmacdo**
Changes can be previewed:
https://asmacdo.github.io/pulpproject.org/2018/04/25/beta-release/

### CODE REVIEW

File: _posts/2018-04-25-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.0 Core Beta Release
```
Comment (bmbouter) [inline _posts/2018-04-25-beta-release.md]: What about adding a '1' on the end to suggest it's numbered and the first.

File: _posts/2018-04-25-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+author: Austin Macdonald
+tags:
+  - release
+  - 3.0
+---
+
+## pulpcore 3.0 beta
```
Comment (bmbouter) [inline _posts/2018-04-25-beta-release.md]: What about adding a '1' on the end to suggest it's numbered and the first.

File: _posts/2018-04-25-beta-release.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Promotion and rollback are fast, synchronous calls.
+- Auto-generated, always up-to-date [API reference
+  documentation](https://docs.pulpproject.org/en/3.0/nightly/integration-guide/rest-api/index.html#pulpcore-rest-api)
+
+
+## What to expect
+
+#### Weekly Releases to PyPI
+
+Releases will continue to be announced on the lists.
```
Comment (bmbouter) [inline _posts/2018-04-25-beta-release.md]: Can we link to our two packages on PyPI somehow?   pulpcore and pulpcore-plugin

https://pypi.org/project/pulpcore/
https://pypi.org/project/pulpcore-plugin/

Comment (asmacdo) [reply _posts/2018-04-25-beta-release.md]: How about "Releases of [pulpcore](link) and [pulpcore-plugin](link) will continue to be announced on the lists.

File: _posts/2018-04-25-beta-release.md (hunk 1 above)
Comment (asmacdo) [reply _posts/2018-04-25-beta-release.md]: "Beta Release 1" or "Beta 1 Release"?

Comment (bmbouter) [reply _posts/2018-04-25-beta-release.md]: Beta 1 Release

File: _posts/2018-04-25-beta-release.md (hunk 3 above)
Comment (bmbouter) [reply _posts/2018-04-25-beta-release.md]: Looks great, ty

**Pull request merged**